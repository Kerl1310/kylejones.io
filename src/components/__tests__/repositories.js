import React from 'react';
import renderer from 'react-test-renderer';

import Repositories from '../repositories';

describe('Repositories', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Repositories />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
