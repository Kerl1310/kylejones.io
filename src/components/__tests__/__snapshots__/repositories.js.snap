// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repositories renders correctly 1`] = `
<div
  className="sc-bwzfXH cPAiWF"
>
  <h2>
    Latest repositories on Github
  </h2>
  <div
    className="repositories__loader"
  >
    <div
      className="sc-bdVaJa lbHbBB"
    >
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
